bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/internal/dto/MQTTProtocol.java

```diff
@@ -155,6 +194,7 @@ public CleanSchedule(Schedule schedule) {
         public Integer batPct;
         // "bin":{"present":true,"full":false}
         public BinStatus bin;
+        public BbSwitch bbswitch;
```

Comment:
Could you also add a comment above the element with an example of contents, like i did ? It's nice to have an idea of what the data look like. For instance my robot doesn't have these items.

Reply:
Should I add the comment or should I remove the line? I added the BbSwitch, because I saw it in the data, but I do not need it for the new command.

Reply:
Are you planning to do something about these values, like providing them as channels / properties?
If not, you can remove them for the sake of code cleanliness. I am using slightly lower-level parsing call, which does not make sure that all of the json has been consumed. So it's OK to have some unhandled fields there.

Reply:
I had planned to make channels for it. However, I am not sure if these channels would be useful. And even if, the class is quickly written again ;-)
So I removed the field and also the BbSwitch class. Furthermore I added a comment to the new field "lastCommand".
Thank you very much for your suggestions.